modules/images/webp-uploads/load.php

```diff
@@ -285,3 +288,36 @@ function webp_uploads_remove_sources_files( $attachment_id ) {
 }
 
 add_action( 'delete_attachment', 'webp_uploads_remove_sources_files', 10, 1 );
+
+/**
+ * Filter on `wp_get_missing_image_subsizes` acting as an action for the logic of the plugin
+ * to determine if additional mime types still need to be created.
+ *
+ * @since n.e.x.t
+ *
+ * @see   wp_get_missing_image_subsizes()
+ *
+ * @param array[] $missing_sizes Associative array of arrays of image sub-size.
+ * @param array   $image_meta    The metadata from the image.
+ * @param int     $attachment_id The ID of the attachment.
+ *
+ * @return array[] $missing_sizes Associative array of arrays of image sub-size.
+ */
+function webp_uploads_wp_get_missing_image_subsizes( $missing_sizes, $image_meta, $attachment_id ) {
+	$trace = array();
+	// Only setup the trace if we no longer have more sizes.
+	if ( empty( $missing_sizes ) ) {
+		$trace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS, 10 );
```

Comment:
Also, might be good to add an inline doc explaining why we use `debug_backtrace` since it is an odd thing to see in code, also that it would not be required for core.

Reply:
Great idea, just added some comments.

Reply:
10 is used to allow for some flexibility, since other plugins may cause the function to be at a slightly higher index than the expected 5 in the stack. E.g. if you use a profiling plugin that hooks into every hook, the indexes here may shift.

+1 on explaining this in a detailed comment. I also suggested a few iterations on that comment in my review.